EventPlanning.BL/EventManager.cs

```diff
@@ -64,16 +68,47 @@ public PartialEvent GetPartialEvent(int eventId)
             return partialEvent;
         }
 
-        private int GetMaxParticipants(int id)
+        /// <summary>
+        /// Returns max participants according to event activities
+        /// </summary>
+        /// <param name="eventId"></param>
+        /// <returns></returns>
+        public int GetMaxParticipants(int eventId)
         {
-            throw new NotImplementedException();
+            var eventActivities = GetEventActivities(eventId);
+
+            if (eventActivities == null)
+            {
+                return 0;
+            }
+
+            return eventActivities.Select(ea => ea.MaxParticipants).Sum();
         }
 
+        /// <summary>
+        /// Returns username from user service
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <returns></returns>
         public string GetUserName(int userId)
         {
-            throw new NotImplementedException();
+            var client = new HttpClient();
+            var serializer = new DataContractJsonSerializer(typeof(int));
+            var jsonInString = JsonConvert.SerializeObject(userId);
+            var url = new StringBuilder();
+            url.AppendFormat("http://localhost:5011/api/users/{0}/partial", userId);
+            var response = client.GetAsync(url.ToString()).Result;
+            var responseContent = response.Content.ReadAsStringAsync().Result;
+            var result = JsonConvert.DeserializeObject<PartialUser>(responseContent);
```

Comment:
Maybe we could write an utility for these calls that can be reused?
Create a separate project for Utilities or a cs file (maybe this might be useful in another project?)